journal/202306.md

```diff
@@ -0,0 +1,861 @@
+- Updated the voting wallet to support [DCP-12](https://github.com/decred/dcrtest/pull/12) block reward split semantics on simnet.
+- Updated the voting wallet to support [DCP-11](https://github.com/decred/dcrtest/pull/14) proof of work semantics on simnet.
+- Updated to use latest released [Decred module](https://github.com/decred/dcrtest/pull/15) dependencies and latest [main dcrd module](https://github.com/decred/dcrtest/pull/16).
+
+{ _omitted minor PRS: 3139, 3140, 3141, dcrtest#13_ }
+
+{ _image candidate: [Initial Sync Total Bytes Used Comparison](https://matrix.to/#/!zefvTnlxYHPKvJMThI:decred.org/$mbJD22XpiecybtWfRIhFRTawODG9wQujjoKSyOqegI0) - what are these "Total Bytes"? memory?_ }
+
+
+### dcrwallet
+
+_[dcrwallet](https://github.com/decred/dcrwallet) is a wallet server used by command-line and graphical wallet apps._
+
+Changes that were included in the new 1.8.0 release:
+
+- Ensure that the wallet can correctly validate blocks mined with either the [original BLAKE-256 or the upcoming BLAKE3](https://github.com/decred/dcrwallet/pull/2240) proof of work hashes, and updated the vote version.
+- Updated [dcrd and cspp modules](https://github.com/decred/dcrwallet/pull/2241) to their latest versions. dcrwallet v3.0.0 module has been released for consumers like vspd, DCRDEX, or Politeia.
+- Mixed ticket purchases now have a [random 20-60s delay](https://github.com/decred/dcrwallet/pull/2244) before the wallet publishes each transaction to the mempool. This trickle effect will improve privacy by making it much harder for anyone surveilling the mempool to link tickets together based on published timestamp.
+
+The following work has been merged in `master` towards future releases:
+
+- Removed mention of legacy [ticket revocations](https://github.com/decred/dcrwallet/pull/2245). { _group or drop (small)_ }
+- Updated [.gitignore](https://github.com/decred/dcrwallet/pull/2246) to ignore test databases. { _group (internal/dev)_ }
+- Updated the [README](https://github.com/decred/dcrwallet/pull/2247) for people building from source.
+- Bumped the major module version of the in-dev [dcrwallet module](https://github.com/decred/dcrwallet/pull/2248) to v4 to prepare for future changes. { _group (internal/dev)_ }
+- [Updated tests](https://github.com/decred/dcrwallet/pull/2250) to account for Go 1.21 changes. { _may group (tests)_ }
+- Dropped [noisy TSpend logs](https://github.com/decred/dcrwallet/pull/2251) in SPV mode that were used for development. { _group (too small)_ }
+- Removed an [unneeded test](https://github.com/decred/dcrwallet/pull/2253) for the database upgrade code. { _group (tests)_ }
+- Removed a deprecated [`AddressReuse` config](https://github.com/decred/dcrwallet/pull/2258) option from the wallet. { _1. add that it was removed from use earlier but was left in code for compatibility; 2. optional: find an earlier removal/deprecation commit and explain what this config did_ }
+
+Works to consolidate VSP code in the vspd repository:
```

Comment:
For some colour you could mention that the end goal of these changes is to extract the VSP client out of dcrwallet and into vspd so that it can be reused in dcrdex, enabling dcrdex to purchase tickets and register them with a VSP.

Reply:
Good idea. I [added](https://github.com/xaur/decred-news/commit/b5f72ff) your copy and extended it a bit. Thanks.